Dockerfile

```diff
@@ -1,27 +1,18 @@
 FROM docker.io/puckel/docker-airflow
 
-LABEL authors="Gordon Inggs"
+LABEL authors="Gordon Inggs and Riaz Arbi"
 
-# Changing back to root to install a few things
+# Changing back to root
 USER root
+RUN echo AIRFLOW_VERSION
+ARG AIRFLOW_VERSION=1.10.3
 
-# Utility packages
+# Adding utility packages
 RUN set -ex && \
   apt-get update -yqq && \
   apt-get upgrade -yqq && \
   apt-get install -yqq \
-  vim \
-  nano \
-  htop \
-  bash \
-  wget \
-  apt-utils \
-  git \
-  sudo
-
-# Installing Docker Python bindings
-RUN DEBIAN_FRONTEND=noninteractive \
-  pip3 install docker
```

Comment:
Note: this merge will presumably break ability to use this image for the python-docker operator use-case

Reply:
Indeed - the name of the repo is now fairly misleading. 

Before the use case was running airflow tasks via Docker, now it's via Kubernetes.